test/scala/com/github/johnynek/bazel_deps/ParseTestCasesTest.scala

```diff
@@ -98,4 +98,42 @@ dependencies:
 """
     roundTripsTo(input, output)
   }
+
+   test("Do not collapse when incompatible") {
+    val input = """
+dependencies:
+   com.twitter:
+    chill:
+      lang: scala
+      version: "0.8.4"
+    chill-algebird:
+      lang: scala
+      version: "0.8.4"
+      exports:
+        - "com.twitter:chill"
+    chill-scrooge:
+      lang: scala
+      version: "0.8.4"
+      exports:
+        - "com.twitter:chill"
+"""
+    val output = """dependencies:
+  com.twitter:
+    chill:
+      lang: scala
+      version: "0.8.4"
+    chill-algebird:
+      exports:\u0020
+        - "com.twitter:chill"
+      lang: scala
+      version: "0.8.4"
+    chill-scrooge:
+      exports:\u0020
```

Comment:
these suck, but that is a bigger issue: remove trailing whitespace in the generated docs.

Reply:
Yeah not a big deal, could do it as post-processing if its hard to fit into the code here too i imagine?  (In one of the projects i have i'm just doing a regex against the file after the formatter runs to strip this)

Reply:
yeah, I want to fix this. I'll do it in a separate issue.

There is a new method in paiges which should do this (`.renderTrim`) but I found a bug in it when I tried to use it.